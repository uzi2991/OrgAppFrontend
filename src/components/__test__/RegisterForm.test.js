import React from "react";
import renderer from "react-test-renderer";
import "@testing-library/jest-dom";
import RegisterForm from "../auth/RegisterForm";

test("LoginForm snapshot", () => {
    const registerFormSnapshot = renderer.create(
        <RegisterForm />
    ).toJSON();

    expect(registerFormSnapshot).toMatchSnapshot();
})