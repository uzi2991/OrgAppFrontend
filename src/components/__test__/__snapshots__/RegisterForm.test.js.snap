// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegisterForm snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  <div
    className="login-fieldset"
  >
    <div
      style={
        Object {
          "display": "block",
        }
      }
    >
      <input
        className="sidebar-input border--gray border--onHoverBlue"
        name="first_name"
        placeholder="First Name"
        type="text"
      />
      <input
        className="sidebar-input border--gray border--onHoverBlue"
        name="last_name"
        placeholder="Last Name"
        type="text"
      />
    </div>
    <div
      style={
        Object {
          "display": "none",
        }
      }
    >
      <input
        className="sidebar-input border--gray border--onHoverBlue"
        name="email"
        placeholder="Email"
        type="email"
      />
      <input
        className="sidebar-input border--gray border--onHoverBlue"
        name="password"
        placeholder="Password"
        type="password"
      />
    </div>
    <div
      className="buttons"
    >
      <button
        className="btn btn--disabled"
        disabled={true}
        id="next"
        onClick={[Function]}
      >
        Next
      </button>
    </div>
  </div>
</form>
`;
